### ISSUE TITLE
Make a release

### ISSUE THREAD

**Issue by Ekleog-NEAR** [near/wasmer#151]
We should make a wasmer release.

This is currently blocked on:
- Us making a proper nearcore release (latest releases have not been very stable)
- @gmilescu giving the greenlight for https://github.com/near/node-docs/pull/55 (so, after we have a proper nearcore release)
- We decide how we want to behave when wasmer changes behavior between one version and the next

Then we can make a release.

Once it’s done, we should be able to merge wasmer into the nearcore repository to simplify development

**Comment by jakmeier** [comment]
> We decide how we want to behave when wasmer changes behavior between one version and the next

Sadly we didn't have time to discuss this today in the meeting, I guess we can wait until next week.

But just to clarify, do we expect this update to be a non-protocol update?
- If yes, how can we be sure?
- If no, well then I guess we have to talk about how we support replayability and such, which we could do in next week's meeting.

**Comment by Ekleog-NEAR** [comment]
It can definitely wait until next week! We’re blocked on quite a lot of other things before being able to make a release anyway :)

The questions you have are all good questions, and especially the wasmparser bump makes me think this should be a protocol update. In addition to that the next thing I’ll be working on is trying to integrate @nagisa’s work on finite-wasm into wasmer, which if landed before the next version can happen would very much definitely be a protocol update. (But then the good thing is it’d be so big that it’d deserve having a "wasmer3")

Hence why I’ve been putting discussing what we want to do upon wasmer changes on the sync agenda for a few weeks, but didn’t push it more as we still have lots of time before being unblocked by other things anyway 😄

**Comment by akhi3030** [comment]
As per https://github.com/near/node-docs/pull/55#issuecomment-1344290277, this is currently blocked till the next nearcore release.

**Comment by Ekleog-NEAR** [comment]
This is currently being replaced by work in progress in near/nearcore#8323, vendoring wasmer into nearcore

**Comment by nagisa** [comment]
Closing as `near-vm` has been merged into `nearcore`.

**Comment by nagisa** [comment]
Bad bot!

**Comment by nagisa** [comment]
Go. AWAY!
